core/src/test/scala/unit/kafka/server/RequestQuotaTest.scala

```diff
@@ -217,9 +218,9 @@ class RequestQuotaTest extends BaseRequestTest {
           new requests.FindCoordinatorRequest.Builder(FindCoordinatorRequest.CoordinatorType.GROUP, "test-group")
 
         case ApiKeys.JOIN_GROUP =>
-          new JoinGroupRequest.Builder("test-join-group", 10000, "", "consumer",
+          new JoinGroupRequest.Builder("test-join-group", 200, "", "consumer",
             List( new JoinGroupRequest.ProtocolMetadata("consumer-range",ByteBuffer.wrap("test".getBytes()))).asJava)
```

Comment:
Also, as discussed offline, there are some unused `builder` fields in this class. I had removed them in a separate PR, but it makes more sense to do it here, probably. As part of that, we can simplify the code a little. Here's what I had done (ignore the `Resource => _, ResourceType => _` change):

https://github.com/ijuma/kafka/commit/49f4b0cf37f8ca396e6e90f015299d12d211932b